tt_metal/impl/dispatch/cq_commands.hpp

```diff
@@ -197,11 +196,16 @@ struct CQDispatchWritePackedMulticastSubCmd {
     uint32_t num_mcast_dests;
 } __attribute__((packed));
 
+constexpr uint32_t CQ_DISPATCH_CMD_PACKED_WRITE_LARGE_FLAG_NONE              = 0x00;
+constexpr uint32_t CQ_DISPATCH_CMD_PACKED_WRITE_LARGE_FLAG_UNLINK            = 0x01;
 struct CQDispatchWritePackedLargeSubCmd {
     uint32_t noc_xy_addr;
     uint32_t addr;
     uint16_t length;          // multiples of L1 cache line alignment
     uint16_t num_mcast_dests;
+    uint8_t flags;
```

Comment:
I think num_mcast_dsts can be a uint8_t and we can pack flags in, right?

Reply:
Ah yes, we have it as uint8_t in some other places too so should be fine to change.